build.gradle

```diff
@@ -10,7 +10,7 @@ ext.hadoopBinaries = "${rootDir}/hadoop-binaries".toString()
 ext.javaDriverVersion = '3.2.1'
 ext.hiveVersion = '1.2.1'
 ext.pigVersion = '0.15.0'
-ext.hadoopVersion = '2.7.2'
+ext.hadoopVersion = '1.2.1'
```

Comment:
@xardazz - Isn't it possible to provide gradle the hadoopVersion on compile time? This should be a variable, not a constant.. no?